src/zygote.jl

```diff
@@ -55,11 +73,13 @@ ZygoteRules.@adjoint function getindex(VA::AbstractVectorOfArray, i::Union{Int,A
   function AbstractVectorOfArray_getindex_adjoint(Δ)
     Δ′ = zero(VA)
     Δ′[i,j...] = Δ
-    (Δ′, i,map(_ -> nothing, j)...)
+    @show Δ′

```

Comment:
```suggestion
```

Reply:
Sorry, very WIP!

Reply:
oh no worries, still worth the downstream tests